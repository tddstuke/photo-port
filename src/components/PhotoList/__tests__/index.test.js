import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import PhotoList from "..";

afterEach(cleanup);

describe("PhotoList is rendering", () => {
  it("renders", () => {
    render(<PhotoList />);
  });
});

describe("matches snapshot", () => {
  const { asFragment } = render(<PhotoList />);
  expect(asFragment()).toMatchSnapshot();
});
