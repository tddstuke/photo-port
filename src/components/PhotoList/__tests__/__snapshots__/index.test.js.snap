// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
    />
  </div>
</DocumentFragment>
`;
